bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
```

Comment:
Did you mean to add this krun config file in this PR?

Reply:
Well I didn't want to switch back and forth after I started iterating on it and it is a overlay file anyway.